File: codes/java/Code_2_02_2.java
import java.util.*;

class Code_2_02_2 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int a = sc.nextInt(); // a を入力する
		int b = sc.nextInt(); // b を入力する
		System.out.println(a & b); // a AND b の値を出力する
		System.out.println(a | b); // a OR b の値を出力する
		System.out.println(a ^ b); // a XOR b の値を出力する
	}
};

File: codes/cpp/Code_2_02_2.cpp
#include <iostream>
using namespace std;

int main() {
	int a, b;
	cin >> a >> b; // a と b を入力する
	cout << (a & b) << endl; // a AND b の値を出力する
	cout << (a | b) << endl; // a OR b の値を出力する
	cout << (a ^ b) << endl; // a XOR b の値を出力する
	return 0;
}